CONTRIBUTING.md

```diff
@@ -37,6 +37,8 @@ Below is a short excerpt of some points in there:
 
     The old config generation system used impure shell scripts and could break in specific circumstances (see #1234).
 
+  In case of adding yourself as maintainer, do it in a separated commit (no need for a Pull Request), with message `maintainers: add <handle>`. See also ([the section of manual about meta.maintainers](https://nixos.org/manual/nixpkgs/stable/#var-meta-maintainers).
```

Comment:
```suggestion
  When adding yourself as maintainer, in the same pull request make a separate commit with the message `maintainers: add <handle>`.
  See [Nixpkgs Maintainers](../maintainers/README.md) for details.
```

Given the information will be expanded in https://github.com/NixOS/nixpkgs/pull/250344, we should rather link to that file. We can still add information on NixOS contributions later.